packages/camera/camera/example/ios/Runner/main.m

```diff
@@ -8,6 +8,8 @@
 
 int main(int argc, char *argv[]) {
   @autoreleasepool {
-    return UIApplicationMain(argc, argv, nil, NSStringFromClass([AppDelegate class]));
+    BOOL isTesting = NSClassFromString(@"XCTestCase") != nil;
+    return UIApplicationMain(argc, argv, nil,
+                             isTesting ? nil : NSStringFromClass([AppDelegate class]));
```

Comment:
The reason for doing this is not at all obvious in the context of this code; please add an explanatory comment so that it's less mysterious to someone reading this.